tensorflow_addons/metrics/r_square.py

```diff
@@ -0,0 +1,44 @@
+import tensorflow as tf
```

Comment:
You need to add the license and future imports like [here](https://github.com/AakashKumarNain/addons/blob/0c31db81ac4d69d63e9fcf0a79aae9feed8e2176/tensorflow_addons/metrics/cohens_kappa.py#L1)